test/functional/fixtures/auth/test.js

```diff
@@ -0,0 +1,13 @@
+describe('Basic and NTLM authentications', function () {
+    it('Try basic authentication with wrong credentials', function () {
```

Comment:
Usually  we use `Should ....(something to do)` in test names 
look at https://github.com/DevExpress/testcafe/blob/master/test/functional/fixtures/api/es-next/click/test.js#L6